pages/partials/uploadThumbnailForm.ejs

```diff
@@ -0,0 +1,49 @@
+<form name="upload-thumbnail-form" class="needs-validation" method="POST" enctype="multipart/form-data" novalidate>
+    <div class="container p-0 mb-4">
+        Upload an image of type jpg, jpeg, png, svg, or gif.
```

Comment:
Minor stylistic nit: can we wrap the file extensions with `<code>` tags (or better yet, `<code class="user-output">`)